src/components/header.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import './header.css';
+import '../styles.css';
 
 const Header = () => (
   <div className="header">
     <h1>Bookstore CMS</h1>
-    <Link to="/">Books</Link>
-    <Link to="/categories">Categories</Link>
+    <Link className="BooksH" to="/">Books</Link>
+    <Link className="CategoriesH" to="/categories">Categories</Link>
   </div>
```

Comment:
- [ ] You will notice there is a profile-like image at the right hand of the template design, kindly add it to yours too. Also, I would encourage you to style the books and categories links to fit in the template as they are close to each other in the template and both are close to the logo unlike what is in yours at the moment.

### Your design
![image](https://github.com/Roland-Ntwali/Bookstore/assets/86200356/d77f915a-dc6b-4991-9182-39f9afa71890)

### Design template
![image](https://github.com/Roland-Ntwali/Bookstore/assets/86200356/05d160cf-c890-4be7-9248-43a9f8295344)